legal-api/src/legal_api/resources/business/business_directors.py

```diff
@@ -55,10 +55,22 @@ def get(identifier, director_id=None):
                 del director_json['mailingAddress']
             res.append(director_json)
 
+        # party role code to replace director
+        party_list = []
+        active_directors = PartyRole.get_active_directors(business.id, end_date)
+        print(active_directors)
+        for director in active_directors:
+            print(director)
+            director_json = director.json
+            print(director_json)
+            if business.legal_type == 'CP':
+                del director_json['mailingAddress']
+            party_list.append(director_json)
+
         return jsonify(directors=res)
 
     @staticmethod
-    def _get_director(business, director_id=None):
+    def _get_director(business, director_id=None, party_id=None):
```

Comment:
I think this needs to get reworked to find either by director_id or party_id, then handle the return of nothing found, etc.

Reply:
That would be nicer, but since this isn't going to last more than a couple days in our code base I don't think it's worth changing. I've already branched off to make changes for after this PR and in that one this method goes back to just the 1 director_id and searches the party/party_role tables. If it's important though I'll obvs change this